catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/summary.hbt

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type"/>
+    <!-- styles -->
+    <link href="/services/registries/assets/css/bootstrap.min.css" rel="stylesheet"/>
+    <style type="text/css">
+        body {
+        background-color: {
+        {
+            background
```

Comment:
Checked and background-color and color do not really affect the report, modified accordingly